server/src/pages/shotindex/view.js

```diff
@@ -10,8 +10,9 @@ class Head extends React.Component {
   render() {
     return (
       <reactruntime.HeadTemplate {...this.props}>
-        <script src={this.props.staticLink("/static/js/shotindex-bundle.js")} async></script>
-        <link rel="stylesheet" href={this.props.staticLink("/static/css/shot-index.css")} />
+        <script src={ this.props.staticLink("/static/js/wantsauth.js") } />
```

Comment:
This should be conditional on `this.props.deviceId`, like:

    { this.props.deviceId ? null : <script src={ this.props.staticLink("/static/js/wantsauth.js") } /> }

i.e., it's kind of aggressive, and you only wantauth if you don't haveauth.